.changeset/few-plants-strive.md

```diff
@@ -0,0 +1,10 @@
+---
+"@fluidframework/devtools-core": minor
+"__section": feature
+---
+Promote IFluidDevtools APIs from alpha to beta
+
+`IFluidDevtools` may now be imported from `/beta`. For example:
```

Comment:
Let's complete the list of things that can now be imported from beta. Arguably, the functions are more relevant than the interfaces.